HISTORY.rst

```diff
@@ -1,7 +1,29 @@
 =======
 History
 =======
-3.41.0 (2012-08-05)
+
+3.42.0 (2021-08-19)
+-------------------
+Critical Changes
+
+* The ``github_release`` task now requires the ``tag_prefix`` option to be passed. (#2792)
```

Comment:
I might elaborate on this a bit:
```suggestion
* The ``github_release`` task now requires the ``tag_prefix`` option to be passed, because for 2nd-generation packages we can't tell from the version number whether it is a beta or not. We've updated the standard release flows to set the ``tag_prefix`` appropriately, but if you have custom flows using this task you will need to update them. (#2792)
```